src/main/java/com/tripply/Auth/config/ApplicationConfiguration.java

```diff
@@ -46,4 +47,13 @@ AuthenticationProvider authenticationProvider() {
 
         return authProvider;
     }
+
+    @Bean
```

Comment:
Remove this unused bean, as the WebClientBuilder bean is used in the WebClientService class.

Reply:
Resolved